netbout-web/src/main/java/com/netbout/email/EmMessages.java

```diff
@@ -75,11 +81,15 @@
     /**
      * Public ctor.
      * @param org Origin
+     * @param slf Self alias
      * @param pst Postman
      * @param bot Bout we're in
+     * @checkstyle ParameterNumberCheck (4 lines)
      */
-    EmMessages(final Messages org, final Postman pst, final Bout bot) {
+    EmMessages(final Messages org, final String slf,
```

Comment:
@ndbroadbent I'd add `@NotNull` to `slf` since this is used as a base for equality down below.